core/src/main/java/com/netflix/conductor/service/WorkflowBulkServiceImpl.java

```diff
@@ -197,22 +197,26 @@ public BulkResponse delete(List<String> workflowIds, boolean archiveWorkflow) {
         return bulkResponse;
     }
 
-    public BulkResponse removeCorrelatedWorkflows(String correlationId, boolean archiveWorkflow) {
+    public BulkResponse removeCorrelatedWorkflows(
+            String correlationId, boolean archiveWorkflow, boolean isPollProcessing) {
         BulkResponse bulkResponse = new BulkResponse();
         long startTime = System.currentTimeMillis();
         Set<String> result = executionDAOFacade.getWorkflowIdSetByCorrelationId(correlationId);
         long corrDuration = System.currentTimeMillis() - startTime;
```

Comment:
If get is possible `getWorkflowIdSetByCorrelationId`, is remove not possible for entire set ?